orchagent/portsorch.cpp

```diff
@@ -261,7 +266,7 @@ PortsOrch::PortsOrch(DBConnector *db, vector<table_name_with_pri_t> &tableNames)
         if (status != SAI_STATUS_SUCCESS)
         {
             SWSS_LOG_ERROR("Failed to get hardware lane list pid:%lx", port_list[i]);
-            throw runtime_error("PortsOrch initialization failure");
+            throw "PortsOrch initialization failure";
```

Comment:
This is a new change. You may want to resolve conflicts

Reply:
OK，I'll resolve the conflicts